src/handlers/revertPr.ts

```diff
@@ -0,0 +1,32 @@
+import axios from "axios";
```

Comment:
rename file to `revert` because that is the command name too

Reply:
Addressed everything